config.h

```diff
@@ -198,23 +193,29 @@ byte I2CAdress = 22;
 #define NUM_LEDS 48
 #define LEDS_PER_COLUMN 6
 #define COLUMNS 10
-  
+ 
+// Addressible LED Array
+// -1 = no LED in that grid space
+
   int8_t ledMatrix[COLUMNS][LEDS_PER_COLUMN] = {
-    { -1, -1, 23, 24, -1, -1 }, 
-    { -1, 6, 22, 25, 41, -1 }, 
-    { 5, 7, 21, 26, 40, 42, },
-    { 4, 8, 20, 27, 39, 43, }, 
-    { 3, 9, 19, 28, 38, 44, }, 
-    { 2, 10, 18, 29, 37, 45, },
-    { 1, 11, 17, 30, 36, 46, },
-    { 0, 12, 16, 31, 35, 47, },
+    { -1, -1, 23, 24, -1, -1, },
+    { -1,  6, 22, 25, 41, -1, },
+    {  5,  7, 21, 26, 40, 42, },
+    {  4,  8, 20, 27, 39, 43, },
+    {  3,  9, 19, 28, 38, 44, },
+    {  2, 10, 18, 29, 37, 45, },
+    {  1, 11, 17, 30, 36, 46, },
+    {  0, 12, 16, 31, 35, 47, },
     { -1, 13, 15, 32, 34, -1, },
     { -1, -1, 14, 33, -1, -1, } 
   };
 
 
 // Command processing stuff
-#define CMD_MAX_LENGTH 64 // maximum number of characters in a command (63 chars since we need the null termination)
+// maximum number of characters in a command (63 chars since we need the null termination)
```

Comment:
Don't like the extra blank line here.  Would rather keep as was or remove the blank line before the #define since other #defines don't have this (see cmdString below).